companion/src/generaledit/generalsetup.cpp

```diff
@@ -21,6 +21,45 @@
 #include "generalsetup.h"
 #include "ui_generalsetup.h"
 
+// Manage timezones
+// For backward compatibility timezone is stored as two separate values:
+//   timezone = hour value
+//   timezoneMinutes - minute value / 15
+static int8_t timezoneHours[] = { -12, -11, -10, -9, -9, -8, -7, -6, -5, -4, -3, -3, -2, -1 , 0, 1, 2, 3,  3, 4,  4, 5,  5,  5, 6,  6, 7, 8,  8, 9,  9, 10, 10, 11, 12, 12, 13, 14 };
+static uint8_t timezoneMins[] = {   0,   0,   0, 30,  0,  0,  0,  0,  0,  0, 30,  0,  0,  0,  0, 0, 0, 0, 30, 0, 30, 0, 30, 45, 0, 30, 0, 0, 45, 0, 30,  0, 30,  0,  0, 45,  0,  0 };
+
```

Comment:
int and unsigned int for Companion as not worried about memory so much and no need to cast to avoid some compiler warnings

Reply:
I was trying to keep the code consistent with the firmware, not save space.

I don't get any warnings, where are you seeing them?

Reply:
Okay missed the code consistency angle and perfectly reasonable justification. However the coding for Companion has been historically to code agnostically where possible so it compiles against all supported OSes and bits without change and only as a last resort resort to ifdefs and custom typedefs.
I didn't get a warning but when I see something out of the Companion ordinary I think it is worth asking why.